keyboards/boardsource/3x4/config.h

```diff
@@ -6,7 +6,7 @@
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xF7E0
-#define PRODUCT_ID      0x0412
+#define PRODUCT_ID      0x0413
```

Comment:
Based on `3x4`, obviously.

```suggestion
#define PRODUCT_ID      0x0304
```

Reply:
happy to make this change - are product id's unique across the project? Or just across a vendor?